hack/add-cluster.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e
+
+root=$(dirname "${BASH_SOURCE[0]}")
+bash $root/install-dev-deps.sh
```

Comment:
I would just leave that script independent.
People can use it if they want to, but don't have to install things through the script. I understand the script checks whether things are already installed, but i'd prefer the tool to tell me something is missing and suggest a script to install it rather than doing it for me

I'm a huge brew user for instance, and would prefer to track it there rather than curling it from github

Reply:
Yeah, I was thinking among the same lines. I've decided to keep it like this for simplicity and have opted for curling stuff instead of relying on brew to make it more cross platformy. In the end one of my intention is to leverage these scripts for github actions e2e testing against actual kind clusters.

The other reason why I used curls instead of brew is because I got burned on installing the brew version of kubebuilder which doesn't come with etcd so the BDD tests were failing.

I propose that:
* We separate the scripts and add instructions on how to run them
* We keep curling for now for simplicity sake and rethink whether we want to offer a brew alternative for OSX once we have resolved the hurdles of installing this in the CI env.

Reply:
That sounds good; i wasn't suggesting by all means enforce brew or anything. My point was to not force the installation.